Certified SAIL Extension (CSE) Examples/mapField/appian-extension.xml

```diff
@@ -6,8 +6,22 @@
     <version>1.0</version>
   </extension-info>
 
-  <component name="Google Map Component" key="google.map">
-    <ruleName>mapField</ruleName>
+  <component ruleName="mapField" version="1.0.0">
+    <sdk-version>2.0.0</sdk-version>
+    <iconFile>google-maps.svg</iconFile>
+    <parameter>
+      <name>location</name>
+      <category>input-only</category>
+      <type>Text</type>
+    </parameter>
+    <parameter>
+      <name>pin</name>
+      <category>event</category>
+    </parameter>
+    <html-entry-point>index.html</html-entry-point>
+  </component>
+  <component ruleName="mapField" version="2.0.0">
```

Comment:
I don't think we need to include an example of using the CSE version here - both `mapField` version 1.0.0 & version 2.0.0 are identical